lib/rubocop/cop/salsify/rails_application_mailer.rb

```diff
@@ -16,8 +16,9 @@ module Salsify
       #  class Yugo < ActionMailer::Base
       #    ...
       #  end
-      class RailsApplicationMailer < Cop
+      class RailsApplicationMailer < RuboCop::Cop::Base
```

Comment:
Using `auto_correct` without this inheritence and mixin is deprecated and throws an error these days.